client/src/app/deck/deck.tsx

```diff
@@ -132,8 +203,29 @@ const Deck = ({setMenuSection}: Props) => {
 					<button type="button" onClick={saveDeck}>
 						Save
 					</button>
-					<button type="button" onClick={loadDeck}>
-						Load
+
+					<select
```

Comment:
Now that we also have the imports/Exports there isn't enough space to fit everything in on one line. Not sure how to handle that.